app/controllers/shipit/webhooks_controller.rb

```diff
@@ -16,7 +16,13 @@ def create
     private
 
     def verify_signature
-      head(422) unless Shipit.github.verify_webhook_signature(request.headers['X-Hub-Signature'], request.raw_post)
+      verified = Shipit.github_organizations.map do |gh_org|
```

Comment:
Github webhooks should include the `repo_owner` (org) field. Did you consider using that instead of iterating over available organizations?

Reply:
Good catch! I've gone ahead and changed this. I followed the same approach as above in `github_sync_job.rb`. The tests for `webhooks_controller.rb` don't define a stack. You are absolutely right though. In real flows one should always be defined. I don't know if this is the PR that we should clean up/fix those tests though.